package.json

```diff
@@ -51,14 +51,18 @@
     "chai-jest-snapshot": "^2.0.0",
     "cross-env": "^7.0.3",
     "mocha": "^9.2.2",
-    "monocart-coverage-reports": "^2.8.3",
     "standard": "^16.0.4",
     "ts-node": "^10.7.0",
     "typescript": "^5.0.0"
   },
-  "c8Requirements": {
+  "peerDependencies": {
```

Comment:
I believe this will install by default, and what I'm trying to avoid is forcing an 800kb for people who are not experimenting with monocart.

Reply:
It won't, I promise! As an example, I did `npm install typechat`:

```console
$ npm install typechat

added 1 package, and audited 2 packages in 1s

found 0 vulnerabilities
$ ls node_modules
typechat
$ cat package-lock.json
{
  "name": "typechattest",
  "version": "1.0.0",
  "lockfileVersion": 3,
  "requires": true,
  "packages": {
    "": {
      "name": "typechattest",
      "version": "1.0.0",
      "license": "ISC",
      "dependencies": {
        "typechat": "^0.1.0"
      }
    },
    "node_modules/typechat": {
      "version": "0.1.0",
      "resolved": "https://registry.npmjs.org/typechat/-/typechat-0.1.0.tgz",
      "integrity": "sha512-1QeypvhyoXz/1DssDq8NbIzRHX3V20b+TWfrO7t4wd3o8OygcQO3Zo9MRNSBwK0Pyw+Fw6MmE/hOFi372gpvzA==",
      "workspaces": [
        "./",
        "./examples/*"
      ],
      "engines": {
        "node": ">=18"
      },
      "peerDependencies": {
        "typescript": "^5.3.3",
        "zod": "^3.22.4"
      },
      "peerDependenciesMeta": {
        "typescript": {
          "optional": true
        },
        "zod": {
          "optional": true
        }
      }
    }
  }
}
```

Reply:
(This is what `peerDependenciesMeta` with `optional` is intended to handle.)

Reply:
Huh, cool I will test too.